src/Kmd.Logic.Cvr.Client/CvrClient.cs

```diff
@@ -92,6 +92,72 @@ public async Task<IList<CvrProviderConfigurationModel>> GetAllCvrConfigurationsA
             return await client.GetAllCvrConfigurationsAsync(subscriptionId: this.options.SubscriptionId).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Get the production units of a company from the CVR register.
+        /// </summary>
+        /// <param name="cvr">The Production unit number.</param>
+        /// <returns>The list of production units of the company.</returns>
+        /// <exception cref="ValidationException">Missing cvr number.</exception>
+        /// <exception cref="SerializationException">Unable process the service response.</exception>
+        /// <exception cref="LogicTokenProviderException">Unable to issue an authorization token.</exception>
+        /// <exception cref="CvrConfigurationException">Invalid CVR configuration details.</exception>
+        public async Task<IList<ProductionUnit>> GetProductionUnitsAsync(string cvr)
+        {
+            var client = this.CreateClient();
+            using (var response = await client.GetProductionUnitByCvrWithHttpMessagesAsync(
+                                subscriptionId: this.options.SubscriptionId,
+                                cvr: cvr,
+                                configurationId: this.options.CvrConfigurationId).ConfigureAwait(false))
+            {
+                switch (response.Response.StatusCode)
+                {
+                    case System.Net.HttpStatusCode.OK:
+                        return (IList<ProductionUnit>)response.Body;
+
+                    case System.Net.HttpStatusCode.NotFound:
+                        return null;
```

Comment:
Normally you'd return an empty list in this case? What does NotFound mean?

Reply:
Cvr not exists?